### PR TITLE
feat(admin): occ twofactor:cleanup/disable

### PR DESCRIPTION
**Pull request by ChristophWurst**
### ☑️ Resolves

* Ref https://github.com/nextcloud/documentation/issues/11170

### 🖼️ Screenshots

image

### CODE REVIEW

**Review by joshtrichards — APPROVED** [review]

File: admin_manual/configuration_server/occ_command.rst [hunk 1]
```diff
@@ -1729,13 +1731,13 @@ In the case of a user losing access to the second factor (e.g. lost phone with
 two-factor SMS verification), the admin can try to disable the two-factor
```
Comment (joshtrichards) [inline admin_manual/configuration_server/occ_command.rst:1731]: Maybe we go ahead and add the full command list to the top of the *Two-factor authentication* section while we're at it - e.g.

The ``twofactorauth`` commands are used to configure/manage various aspects of two-factor authentication::
```
      twofactorauth:cleanup Clean up the two-factor user-provider association of an uninstalled/removed provider  
      twofactorauth:enforce Enabled/disable enforced two-factor authentication                                    
      twofactorauth:enable  Enable two-factor authentication for a user                                           
      twofactorauth:disable Disable two-factor authentication for a user                                          
      twofactorauth:state   Get the two-factor authentication (2FA) state of a user.                              
```

(unfortunately GitHub comments don't let one select lines outside the PR AFAIK so I can't make this a suggestion)

Comment (ChristophWurst) [reply admin_manual/configuration_server/occ_command.rst:1731]: I would like that. But for the sake of a minimal backport, I would do it separately.

**Pull request merged**